### ISSUE TITLE
'property-units' rule doesn't work?

### ISSUE THREAD

**Issue by adam-lynch** [sasstools/sass-lint#1050]
**What version of Sass Lint are you using?** 1.10.2

**Please include any relevant parts of your configuration**

Here is the file (`sassLintRules.js`) containing my rules:

```javascript
// OK this file is actually CoffeeScript in reality but I'm too lazy to convert back to JS :)
disabled = 0
warn = 1
error = 2

module.exports = {
    'border-zero': error
    'brace-style': [
        error
        { 'allow-single-line': false }
    ]
    'class-name-format': [
        error
        { 'convention': 'hyphenatedbem' }
    ]
    'declarations-before-nesting': error
    'empty-line-between-blocks': [
        error
        { 'allow-single-line-rulesets': false }
    ]
    'extends-before-mixins': error
    'force-element-nesting': error
    'force-pseudo-nesting': error
    'function-name-format': [
        error
        { 'convention': 'hyphenatedbem' }
    ]
    indentation: [
        error
        { 'size': 4 }
    ]
    'leading-zero': [
        error
        { 'include': true }
    ]
    'max-file-line-count': error
    'max-line-length': error
    'mixins-before-declarations': error
    'mixins-name-format': [
        error
        { 'convention': 'hyphenatedbem' }
    ]
    'no-attribute-selectors': error
    'no-combinators': warn
    'no-css-comments', disabled
    'no-color-keywords': error
    'no-color-literals': error
    'no-duplicate-properties': error
    'no-empty-rulesets': error
    'no-ids': error
    'no-important': error
    'no-invalid-hex': error
    'no-misspelled-properties': error
    'no-qualifying-elements': error
    'no-trailing-whitespace': error
    'no-trailing-zero': error
    'no-transition-all': error
    'no-universal-selectors': error
    'no-url-domains': error
    'no-vendor-prefixes': error
    'one-declaration-per-line': error
    'property-units': [
        error
        {
            global: ['em', 'px', 'rem']
            // Also tried globally-allowed-units
        }
    ]
    'pseudo-element': error
    'shorthand-values': warn
    'single-line-per-selector': error
    'space-after-colon': error
    'space-after-comma': error
    'space-around-operator': error
    'space-before-bang': error
    'space-before-brace': error
    'space-before-colon': error
    'space-between-parens': error
    'trailing-semicolon': error
    'url-quotes': error
    'variable-name-format': [
        error
        { 'convention': 'hyphenatedbem' }
    ]
    'zero-unit': error
}
```

**What did you do? Please include the actual source code causing the issue.**

I'm using gulp-sass-lint@1.3.2. Simplified example:

```javascript
var gulp = require('gulp');
var sassLint = require('gulp-sass-lint');

gulp.task('default', function(){
  gulp.src('./styles/index.scss')
    .pipe(sassLint({ rules: require('./sassLintRules') })
    .pipe($.sassLint.format())
    .pipe($.sassLint.failOnError())
});
```

`index.scss`:

```scss
$blue: #00f;
$red: #f00;

.loginBox { // sass-lint:disable-line class-name-format
    border-width: 2in;
    outline: 1pt solid $red;

    &:hover {
        outline: 1px solid $blue;
    }
}
```

**What did you expect to happen?**

I'd expect the usage of `in` and `pt` to be flagged.

**What actually happened? Please include any error messages given to you by Sass Lint.**

No errors.

---

Also, while I have you, is `%` always allowed or do I need to list it in the allowed units?

**Comment by adam-lynch** [comment]
Created an issue at gulp-sass-lint just in case: https://github.com/sasstools/gulp-sass-lint/issues/77

**Comment by adam-lynch** [comment]
Nevermind, I'm an idiot. It was a problem with my setup.
